cmd/context/list.go

```diff
@@ -52,20 +56,41 @@ func executeListContext(ctx context.Context) error {
 		return fmt.Errorf("no contexts are available. Run 'okteto context' to configure your first okteto context")
 	}
 
-	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tNamespace\tBuilder\tRegistry\n")
 	ctxStore := okteto.ContextStore()
+
+	var ctxs []okteto.OktetoContext
 	for _, ctxSelector := range contexts {
 		if okCtx, ok := ctxStore.Contexts[ctxSelector.Name]; ok && okCtx.Builder != "" {
 			ctxSelector.Builder = okCtx.Builder
 		}
+		ctxs = append(ctxs, okteto.OktetoContext{Name: ctxSelector.Name, Namespace: ctxSelector.Namespace, Builder: ctxSelector.Builder, Registry: ctxSelector.Registry})
+	}
 
-		if ctxSelector.Name == ctxStore.CurrentContext {
-			ctxSelector.Name += " *"
+	if output == "" {
+		w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
+		fmt.Fprintf(w, "Name\tNamespace\tBuilder\tRegistry\n")
+		for _, ctx := range ctxs {
+			if ctx.Name == ctxStore.CurrentContext {
+				ctx.Name += " *"
+			}
+			fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", ctx.Name, ctx.Namespace, ctx.Builder, ctx.Registry)
+		}
+		w.Flush()
+	} else if output == "json" {
+		ctxRaw, err := json.MarshalIndent(ctxs, "", "\t")
+		if err != nil {
+			return err
+		}
+		fmt.Print(string(ctxRaw))
```

Comment:
If we use new line at the end we can get rid off the '%' at the end of the command
```suggestion
		fmt.Println(string(ctxRaw))
```